### ISSUE TITLE
Change the map from openstreet to HERE

### ISSUE THREAD

**Issue by trumboosahil** [VROOM-Project/vroom#323]
Can we change openstreet map to here map

**Comment by jcoupey** [comment]
TL;DR currently not out-of-the-box, but it could be added with a bit of dev work.

Our optimization solving approach can use any underlying routing layer that's able to provide a matrix of travel times and detailed informations for a route with waypoints.

Then it comes down to writing a wrapper that takes care of the requests. We currently have one for OSRM and one for Openrouteservice (see [routing folder](https://github.com/VROOM-Project/vroom/tree/master/src/routing)). Adapting those to use a remote service is not technically a problem. Also related to #219.

Another take on this is to use the [matrix](https://github.com/VROOM-Project/vroom/blob/master/docs/API.md#input) key in input to feed `vroom` the matrix you've computed by whatever mean suits you.

**Comment by trumboosahil** [comment]
Thanks a lot for your prompt response. [matrix](https://github.com/VROOM-Project/vroom/blob/master/docs/API.md#input ) . I have route matrix of two dimensional array.

Can i use vroom on windows on warp it in REST apis using Java or C#

**Comment by jcoupey** [comment]
> So instead of passing location. I will call here map and get route matrix and pass it to vroom

Correct.

> Can i use vroom on windows on warp it in REST apis using Java or C#

If you go through the issues you'll find reports of some users that managed to setup the project under Windows. You might also consider using https://github.com/VROOM-Project/vroom-docker. 

To expose the API, check out https://github.com/VROOM-Project/vroom-express/.

**Comment by trumboosahil** [comment]
Can you please provide me the sample URL request for matrix as in put

**Comment by trumboosahil** [comment]
What about the vehicles i have to provide the co-ordinates for it
